### ISSUE TITLE
helm template command fails

### ISSUE THREAD

**Issue by cardoe** [nautobot/helm-charts#355]
When using `helm template` locally or via ArgoCD it fails due to the comments posted on #249.

```
error: Error: execution error at (nautobot/templates/secret.yaml:17:4): Existing Nautobot DB secret 'my-existing-secret' not found!
```

But I can confirm the secret is there.

**Comment by m4rg4sh** [comment]
Hi @cardoe,

Thank you so much for your work, we've hit the same issue pretty much on the day you opened your issues.
I've tried your contributed code yesterday and I have found two issues with it:

a) Your reference for `.Values.postgresql.auth.secretsKeys.adminPasswordKey` has a typo which conflicts with the postgresql subchart (secret*s*Keys vs secretKeys) and also references the admin password where instead it should use the user password.

b) Your solution conflicts with the [documented usage of preexisting secrets](https://docs.nautobot.com/projects/helm-charts/en/stable/advanced-features/existing-secrets/). If used as documented in the top half (without specifying nautobot.db.existingSecret) it will actually overwrite any existing password in the referenced secret with empty string.

I've written a fix for both issues and will open a PR for it in a moment.
I've tested it with the postgresql but I think it should just as well work with mariadb or postgresqlha.

Again thank you so much for you work, you saved me a lot of headaches 👍
